fluss-server/src/main/java/com/alibaba/fluss/server/tablet/TabletService.java

```diff
@@ -312,7 +314,38 @@ public CompletableFuture<ListOffsetsResponse> listOffsets(ListOffsetsRequest req
 
     @Override
     public CompletableFuture<InitWriterResponse> initWriter(InitWriterRequest request) {
-        // todo: add authorization for table acl until https://github.com/alibaba/fluss/issues/756.
+        List<PbTablePath> tablePathsList = request.getTablePathsList();
+        if (authorizer != null) {
+            if (tablePathsList == null || tablePathsList.isEmpty()) {
+                throw new UnknownTableOrBucketException(
```

Comment:
should throw `AuthorizationException` here, because `UnknownTableOrBucketException` is a retriable exception which indicates clients to retry the request again.